### DISCUSSION TITLE
Ordering of cards: definitions first, then example sentences

### DISCUSSION THREAD

**Discussion by chrsdk** [st3v3nmw/obsidian-spaced-repetition#968] [Q&A]
I am trying to create a deck in which I put word definitions and example sentences together, all in one note, such as:

word1 ::: definition1

word2 ::: definition2

word3 ::: definition3

This is example sentence for ==word1==.

This is example sentence for ==word2==.

This is example sentence for ==word3==.

Is it possible to have the new cards ordered **sequentially**, so that the definitions will always precede the example sentences?

After that the review process would then go through the review cards **randomly**.

Then, when I add new cards to the deck, it would again present the new cards sequentially. After going through them sequentially, they enter the review queue, after which they are presented randomly. And so on.

Currently, when I add new cards, they are all presented randomly, so I get an example sentence before learning the definition, which doesn't make sense.

Would it be possible to change this behavior?

**Comment by 4Source** [comment]
At the moment there is no way, except for manually switch settings, but I opened a feat #946 request which could be extended to this functionality

**Comment by ronzulu** [comment]
Hi @4Source another idea (instead of more intricate sequencing options) would be to implement something like Anki's learning mode https://docs.ankiweb.net/studying.html

That way in "study phase" cards within a deck would be sequential, and during "review phase" random. (User configurable)

There have been requests for this e.g. [ [FEAT] Learning phase for new/forgotten cards #794 ](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/794) for other reasons

Cheers
Ronny

**Reply by 4Source** [reply]
Did you mean to allow the different settings from feat #946 for two different "phases" sound great to me more flexibility

**Reply by ronzulu** [reply]
Yes that's what I had in mind. The idea of learning phases like in Anki makes sense to me, and I think it might be easier to describe the review ordering to a user than by making #946 more feature rich.

Also, I think the normal sequence would be for cards to go from "study phase" to "review phase". But users would have the ability to change back from review phase to study phase. Supporting that in #946 might be difficult.

**Reply by 4Source** [reply]
Maybe also handle the Cram as a separate phase

**Reply by 4Source** [reply]
But how do you deal with which phase the user is in? Can he choose himself, which I think makes the most sense, but how do you do it intuitively? A solution could be to add like in #987 button to the bottom but instead of one button that says just review the 3 different modes ``study phase``, ``review phase`` and ``cram phase`` 

Here is how I understand those phases
**study phase** 
You repeat only the new cards multiple times till you gained a specific threshold of ease so they get repeated more often if you rate them as hard. 
**review phase**
You review only the due cards once like it is currently 
**cram phase** 
You repeat the cards over and over again
